articles/active-directory/manage-apps/use-scim-to-provision-users-and-groups.md

```diff
@@ -78,7 +78,8 @@ Applications that support the SCIM profile described in this article can be conn
    *Figure 3: Configuring provisioning in the Azure portal*
     
 1. In the **Tenant URL** field, enter the URL of the application's SCIM endpoint. Example: https://api.contoso.com/scim/v2/
-1. If the SCIM endpoint requires an OAuth bearer token from an issuer other than Azure AD, then copy the required OAuth bearer token into the optional **Secret Token** field. 
+1. If the SCIM endpoint requires an OAuth bearer token from an issuer other than Azure AD, then copy the required OAuth bearer token into the optional **Secret Token** field. If this field is left blank, Azure AD includes an OAuth bearer token issued from Azure AD with each request. Apps that use Azure AD as an identity provider can validate this Azure AD-issued token.
```

Comment:
We want to change the Apps to Applications throughout the document.

Reply:
let's fix this as part of a revamp of the doc I'm targeting for July. I'm going to try and interview some ISVs to understand how we can improve / restructure the doc.